### ISSUE TITLE
Option to set network interface to use

### ISSUE THREAD

**Issue by baflo** [marthoc/docker-deconz#52]
Hi and first of all, thank you for the great dockerizing of deconz.

I have it running on a Win10 machine and there in a boot2docker environment with network bridge. The containers have two interfaces, `eth0` is the NAT and `eth1` the bridged network.

Consequently, I have kind of a problem with discovery, as it delivers the NAT IP instead of the local network IP. Is there a way to tell deconz which network interface to use.

PS: I had it running on port 7080, but that didn't work with Home Assistant and neither with OpenHAB. Had to switch back to port 80. Is this a familiar problem?

**Comment by marthoc** [comment]
I don't think there's an option available to tell deCONZ which network interface to use. I guess this isn't specific to Docker as you might see the same problem on a machine with multiple NICs. @manup any ideas?

As far as the port goes as long as you change the environment variables for WEB and WS port, then it should at least work with Home Assistant (though you may need to manually specify the config in configuration.yaml); not sure about OpenHAB.

**Comment by baflo** [comment]
That would be great!

**Comment by marthoc** [comment]
@manup I'm doing some followup on old issues - has this feature been implemented in deconz?

**Comment by nullEuro** [comment]
@marthoc yes. See https://github.com/dresden-elektronik/deconz-rest-plugin/issues/444

**Comment by Mimiix** [comment]
@marthoc is this still valid?
